app/index.html

```diff
@@ -68,6 +68,8 @@
       </div>
     </div>
 
+    <img src="images/loginarrow.png" class="loginarrow" alt="loginarrow" ng-if="!loggedIn">
```

Comment:
actually no ;) ng-if deletes the DOM node when the value is false, so once logged in we don't have to keep the image in RAM